operator/src/main/java/oracle/kubernetes/weblogic/domain/model/CommomEnvVars.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2021, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.weblogic.domain.model;
+
+public class CommomEnvVars {
```

Comment:
Is this for common mount only?  The class name seems to suggest a generic environment variable class

Reply:
+1 Suggest 'CommonMountEnvVars'

Reply:
I was envisioning this class to have all the environment variables which are populated in both the introspector job pod and WebLogic server pods. Currently, we have `IntrospectorJobEnvVars` and `ServerEnvVars` but I didn't find a place to add env variables that are common to both "introspector job" and "server pod". One example of this is WDT_INSTALL_HOME which needs to be populated in all the pods. Is there an existing place for common env variables and I missed it? Thanks.

Reply:
I have changed the class name to CommonMountEnvVars and moved the WDT_INSTALL_HOME env variable to `IntrospectorJobEnvVars` (to be colocated with WDT_MODEL_HOME).